### PR TITLE
FR - Translate intro.md

### PR DESCRIPTION
**Pull request by sven-hash**

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/intro.md [hunk 1]
```diff
@@ -10,27 +10,28 @@ title: Intro
 Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
 :::
```
Comment (Sakrecoer) [inline i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: Thank you very much @sven-hash ! Could you please remove the information message stating "Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page." :)

**Review by Sakrecoer — COMMENTED** [review]
Other than the information message in the header, it looks good to me :) Many thanks!

File: i18n/fr/docusaurus-plugin-content-docs/current/intro.md (hunk 1 above)
Comment (sven-hash) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: Done

Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: Awesome! Thank you @sven-hash !

Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: When i test the site, there are 2 Markdown links that do not render properly.
image

It's very strange; i can't figure out what is wrong with them. It looks like there is a space, but there is no such space in the source file. I will investigate and merge when i figure it out.

Comment (sven-hash) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: There was a space, I've corrected them, sorry about this

Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: ok i was in the wrong branch :D there _is_ a space there. Will fix and merge!

Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/intro.md]: oh! haha! ok great! :)

**Pull request merged**